test/linalg/symmetric.jl

```diff
@@ -3,36 +3,37 @@
 using Base.Test
 
 srand(101)
-debug = false #Turn on for more debugging info
 
-#Pauli σ-matrices
-for σ in map(Hermitian, Any[ eye(2), [0 1; 1 0], [0 -im; im 0], [1 0; 0 -1] ])
+@testset "Pauli σ-matrices" for
+    σ in map(Hermitian, Any[ eye(2), [0 1; 1 0], [0 -im; im 0], [1 0; 0 -1] ])
```

Comment:
If you read from the start of a line, for example `f in (eigfact, eigvals, eig),` doesn't make any sense on its own but based on its indentation it looks like it's the first statement in the block - visually figuring out what's going on there requires backtracking to the end of the previous line to see there's a dangling incomplete `for` there. If the indentation makes it more clear that it's a line continuation rather than a normal statement within the body of the block, and the line break is done after the `in` instead of after the `for`, it's quicker to scan over what the code is doing.